app/routes.py

```diff
@@ -1 +1,245 @@
-from flask import Blueprint
\ No newline at end of file
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import Blueprint, jsonify, make_response, request, abort
+from datetime import datetime
+import requests
+import json
+from dotenv import load_dotenv
```

Comment:
Great work! We need to import `load_dotenv` to set up environment variables in our `.env` file